libexec/pyenv-init

```diff
@@ -166,7 +159,7 @@ function help_() {
       echo
       echo 'export PYENV_ROOT="$HOME/.pyenv"'
       echo '[[ -d $PYENV_ROOT/bin ]] && export PATH="$PYENV_ROOT/bin:$PATH"'
-      echo 'eval "$(pyenv init -)"'
+      echo "eval \"\$(pyenv init - $shell)\""
```

Comment:
You can break single quoting here to insert an envvar: `echo 'eval "$(pyenv init - '$shell')"'`

Reply:
Should be fine to do `'eval "$(pyenv init -'$shell')"'`
Instead of `'eval "$(pyenv init -'"$shell"')"'`

Reply:
Yes, if you want the value of `$shell` to be subject to word split since it isn't quoted.
In this case though, it's supposed to be a single word so it doesn't matter, fine either way.